### PR TITLE
Added option to send to sqs and minor cleanup

### PR DESCRIPTION
**Pull request by sourencho**
note: rewrote the structure of the promises to be cleaner

### CODE REVIEW

File: src/enter.js [hunk 1]
```diff
@@ -6,25 +6,29 @@ var Promise = require('bluebird');
 
 module.exports = function(ipp, commander) {
   return new Promise(function(resolve, reject) {
-    var logger = commander.quiet ? {log: _.noop, error: _.noop} : console;
-    var maxRequests = commander.max ? parseInt(commander.max, 10) : 100;
+    var logger = {log: _.noop, error: _.noop}; // no crawler logging for now
```
Comment (wilsonianb) [inline src/enter.js:9]: Why no logging?

Comment (sourencho) [inline src/enter.js:9]: I need to revisit logging in the crawler itself.
I'm not sure how useful it is.

It would be a separate PR

**Comment by wilsonianb** [comment]
LGTM :+1:
